src/SFA.DAS.AssessorService.Web/Views/Application/Submitted.cshtml

```diff
@@ -1,56 +1,139 @@
+@using System.Diagnostics
+@using SFA.DAS.AssessorService.ApplyTypes
+@using SFA.DAS.AssessorService.Web.ViewModels.Apply
 @model SFA.DAS.AssessorService.Web.ViewModels.Apply.SubmittedViewModel
 @{
     ViewBag.Title = "Application submitted";
     Layout = "_Layout";
 }
 
 <main class="govuk-main-wrapper govuk-main-wrapper--l" id="main-content" role="main">
-    <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
-            <div class="govuk-panel govuk-panel--confirmation">
-                <h1 class="govuk-panel__title">
-                    @if (string.IsNullOrEmpty(Model.StandardName))
-                    {
-                        <text>Stage 1 of your application has been submitted</text>
-                    }
-                    else
-                    {
-                        <text>Stage 2 of your application has been submitted</text>
-                    }
-                </h1>
-                <div class="govuk-panel__body">
-                    @if (!string.IsNullOrEmpty(Model.StandardName))
-                    {
-                        <div class="govuk-!-margin-bottom-4">
-                            <span class="govuk-!-font-size-24">Standard applied for</span>
-                            <br />
-                            <strong>@Model.StandardName</strong>
+
+    @if (Model.ApplicationSubmissionType == ApplicationSubmissionType.WithdrawalFromRegister ||
+         Model.ApplicationSubmissionType == ApplicationSubmissionType.WithdrawalFromStandard)
+      {
+
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <form method="post">
+
+                    <div class="govuk-panel govuk-panel--confirmation">
+                        <h1 class="govuk-panel__title">
+                            Withdrawal notification submitted
+                        </h1>
+
+                        <div class="govuk-panel__body">
+                            Your reference number
+                            <br>
+                            <strong>@Model.ReferenceNumber</strong>
                         </div>
+
+                    </div>
+
+
+                    <h2 class="govuk-heading-m govuk-!-margin-top-6">
+                        What happens next
+                    </h2>
+
+
+                    @if (Model.ApplicationSubmissionType == ApplicationSubmissionType.WithdrawalFromRegister)
+                    {
+                        <p class="govuk-body">
```

Comment:
This part looks the same below so should not be duplicated

Reply:
updated